test/SerialIOTest.cpp

```diff
@@ -1112,8 +1112,7 @@ void test_complex(const std::string & backend) {
     if( backend != "json" ) //! @todo clarify that complex data is not N+1 data in JSON
     {
         Series list{ "../samples/serial_write_complex." + backend, Access::READ_ONLY };
-        if( list.backend() != "ADIOS1" ) // FIXME: "Internal error: Failed to inquire about ADIOS variable during dataset opening"
-            helper::listSeries( list );
+        helper::listSeries( list );
```

Comment:
```
[ADIOS1] Internal error:Failed to inquire about ADIOS variable '/data/0/meshes/CfltCflt' during dataset opening
```

Curious, since it should only have a problem with `Cldbl`, which we don't write.